src/commands/cork_proposal/cellar_v2_5/set_strategist_platform_cut.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{
+    application::APP,
+    cellars,
+    commands::cork_proposal::print_proposal,
+    prelude::*,
+    proto::{
+        cellar_v2_5governance::{Function, SetStrategistPlatformCut},
+        governance_call::Call,
+        CellarV25governance, GovernanceCall,
+    },
+};
+use abscissa_core::{clap::Parser, Command, Runnable};
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls setStrategistPlatformCut() on the target V2.5 cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
+)]
+pub struct SetStrategistPlatformCutCmd {
+    #[clap(short, long)]
+    /// New platform cut proportion for the Strategy Provider between 0 and 1e18 representing 0% and 100% respectively.
+    new_platform_cut: u64,
```

Comment:
Nit: here as well.

Reply:
"cut" seems less specific even if it does match?